config/karma-conf.js

```diff
@@ -34,6 +34,7 @@ module.exports = function(config) {
 			'node_modules/jquery/dist/jquery.js',
 			'tests/utils/common-setup.js',
 			'lib/box.js',
+            'lib/events-jquery.js',
```

Comment:
fixed, do we want to include something like .editorconfig to align text/spaces from other contributors?